client/src/components/HeaderComponent.tsx

```diff
@@ -21,11 +23,21 @@ export const HeaderComponent = (props: HeaderComponentPropsType) => {
     const { title, isBackButton = false, isLogoutButton = false } = props;
     const navigation = useNavigation();
 
+    const goBack = () => {
+        navigation.goBack();
+    };
```

Comment:
why do we need a wrapper for goBack

Reply:
it seemed ugly to me to use goBack without a wrapper, and logout with a wrapper